go.mod

```diff
@@ -22,6 +24,7 @@ require (
 
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v12.1.0+incompatible
+	github.com/aliyun/alibaba-cloud-sdk-go => github.com/aliyun/alibaba-cloud-sdk-go v1.61.112
```

Comment:
The `replace` is not made to force a version.

you just have to set the version in the `require` section.

Reply:
You're right! Sorry for that 😑

Reply:
After thinking a bit more about that, I've remembered that I've read the following thing in the [go module documentation](https://github.com/golang/go/wiki/Modules#when-should-i-use-the-replace-directive).

> `replace` allows the top-level module control over the exact version used for a dependency, such as:
>
> - replace example.com/some/dependency => example.com/some/dependency v1.2.3

WDYT?